scripts/load-backup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -e
+
+# Check arguments
+if [ "$#" -ne 1 ]; then
+  echo "usage: $0 osrd.backup"
+  exit 1
+fi
+
+echo "Note: You should only have your postgres container runnning!"
+
+# Copy needed files to the container
+docker cp "$1" osrd-postgres:/tmp/backup-osrd
+
+# Drop database
+echo Deleting osrd database...
+docker exec osrd-postgres psql -c 'drop database osrd;' > /dev/null
+
+echo Initialzie new database...
+docker exec osrd-postgres psql -f /docker-entrypoint-initdb.d/init.sql > /dev/null
+
+# restoring the backend can partialy fail, and that's sometimes ok
+echo Restore backup...
```

Comment:
I somewhat assumed this script would validate the backup using the hash in the filename
It does not seem to, is it intentional?

Reply:
I added the check